tests/nnfw_api/src/ValidationTestAddSessionPrepared.cc

```diff
@@ -68,6 +77,28 @@ TEST_F(ValidationTestAddSessionPrepared, output_tensorinfo)
   ASSERT_EQ(tensor_info.dims[0], 1);
 }
 
+TEST_F(ValidationTestAddSessionPrepared, neg_await_without_async_run)
+{
+  SetInOutBuffers();
+  ASSERT_EQ(nnfw_await(_session), NNFW_STATUS_ERROR);
+}
+
+TEST_F(ValidationTestAddSessionPrepared, neg_await_twice)
+{
+  SetInOutBuffers();
+  ASSERT_EQ(nnfw_run_async(_session), NNFW_STATUS_NO_ERROR);
+  ASSERT_EQ(nnfw_await(_session), NNFW_STATUS_NO_ERROR);
+  ASSERT_EQ(nnfw_await(_session), NNFW_STATUS_ERROR);
+}
+
+TEST_F(ValidationTestAddSessionPrepared, neg_run_during_async_run)
+{
+  SetInOutBuffers();
+  ASSERT_EQ(nnfw_run_async(_session), NNFW_STATUS_NO_ERROR);
+  EXPECT_EQ(nnfw_run(_session), NNFW_STATUS_ERROR);
+  ASSERT_EQ(nnfw_await(_session), NNFW_STATUS_NO_ERROR);
+}
+
```

Comment:
We can add test for `nnfw_run(_session)` -> `nnfw_await(_session)` without call `nnfw_run_async(_session)`.

Reply:
Added `neg_await_after_sync_run` above.